x-pack/plugins/observability_ai_assistant/public/components/prompt_editor/prompt_editor_natural_language.tsx

```diff
@@ -53,7 +53,6 @@ export function PromptEditorNaturalLanguage({
 
     if (textarea) {
       textarea.focus();
-      textarea.select();
```

Comment:
Why was it there in the first place?

Reply:
For the use case of editing an existing message. The user would click "edit", and the text area would get focus and select the contents.